search.ipynb

```diff
@@ -465,6 +466,12 @@
     "\n",
     "* `path(self)` : This returns a list of all the nodes that lies in the path from the root to this node.\n",
     "\n",
+    "The remaining 4 methods override standards Python functionality for representing an object as a string, the less-than ($<$) operator, the equal-to ($=$) operator, and the \n",
```

Comment:
When you mention the newline character, I would like it if you mentioned that it is the newline character in case people didn't know. Something like this:

"and the newline character ('\n')"

Reply:
Hmm, that's odd. I was not referring to the newline character at all. I meant to refer to `hash()`. I'll change this.